modules/phase_field/src/ics/MultiSmoothSuperellipsoidIC.C

```diff
@@ -95,29 +95,46 @@ MultiSmoothSuperellipsoidIC::computeSuperellipsoidSemiaxes()
 {
   for (unsigned int i = 0; i < _numbub[_gk]; i++)
   {
-    //Vary bubble radius
-    switch (_semiaxis_variation_type)
-    {
-    case 0: //Uniform distrubtion
-      _as.push_back(_semiaxis_a[_gk]*(1.0 + (1.0 - 2.0 * _random.rand(_tid)) * _semiaxis_a_variation[_gk]));
-      _bs.push_back(_semiaxis_b[_gk]*(1.0 + (1.0 - 2.0 * _random.rand(_tid)) * _semiaxis_b_variation[_gk]));
-      _cs.push_back(_semiaxis_c[_gk]*(1.0 + (1.0 - 2.0 * _random.rand(_tid)) * _semiaxis_c_variation[_gk]));
-      break;
-    case 1: //Normal distribution
-      _as.push_back(_random.randNormal(_tid, _semiaxis_a[_gk],_semiaxis_a_variation[_gk]));
-      _bs.push_back(_random.randNormal(_tid, _semiaxis_b[_gk],_semiaxis_b_variation[_gk]));
-      _cs.push_back(_random.randNormal(_tid, _semiaxis_c[_gk],_semiaxis_c_variation[_gk]));
-      break;
-    case 2: //No variation
-      _as.push_back(_semiaxis_a[_gk]);
-      _bs.push_back(_semiaxis_b[_gk]);
-      _cs.push_back(_semiaxis_c[_gk]);
-    }
-
-    if (_as.back() < 0.0) _as.back() = 0.0;
-    if (_bs.back() < 0.0) _bs.back() = 0.0;
-    if (_cs.back() < 0.0) _cs.back() = 0.0;
-
+      switch (_semiaxis_variation_type)
+      {
+      case 0: //Uniform distrubtion
```

Comment:
case statements are indented (and I'd prefer having an empty line separating consecutive case blocks)